scripts/release_ppa.sh

```diff
@@ -3,11 +3,11 @@
 ## This is used to package .deb packages and upload them to the launchpad
 ## ppa servers for building.
 ##
-## If no argument is given, creates a package for the develop branch
-## and uploads it to the ethereum/ethereum-dev ppa.
+## You can pass an branch name as argument to this script (which, if no argument is given,
+## will default to "develop").
 ##
-## If an argument is given, it is used as a tag and the resulting package
-## is uploaded to the ethereum/ethereum ppa.
+## If the gien branch is "release", the resulting package will be uplaoded to
+## ethereum/ethereum PPA, or ethereum/ethereum-dev PPA otherwise.
 ##
 ## The gnupg key for "builds@ethereum.org" has to be present in order to sign
```

Comment:
Side-note: we should document this in our release check list.

Reply:
The release checklist is saying that you must call `release_ppa.sh release` already. I didn't see a reason to document anything else here. (maybe, that _now_ you can also write `release_ppa.sh v0.6.0` iff `v0.6.0` is your release branch)